deploy/examples/import-external-cluster.sh

```diff
@@ -14,6 +14,7 @@ MON_SECRET_CEPH_USERNAME_KEYNAME=ceph-username
 MON_SECRET_CEPH_SECRET_KEYNAME=ceph-secret
 MON_ENDPOINT_CONFIGMAP_NAME=rook-ceph-mon-endpoints
 ROOK_EXTERNAL_CLUSTER_NAME=$NAMESPACE
+ROOK_RBD_FEATURES=${ROOK_RBD_FEATURES:-"layering"}
```

Comment:
If this can be changed, we can have a flag from python script(external script) to update this value?